### ISSUE TITLE
Group vs total weighting

### ISSUE THREAD

**Issue by rb643** [rb643/dti#2]
Currently matrices reflect the creation of group-averaged connectome, accepting a connection if it is present in say, 40-60% of the patients within that specific group. Should it be across the entire sample instead?

**Comment by ElijahMak** [comment]
Yes, was planning to do that especially for the group comparisons!
